src/components/homePage/Countdown.vue

```diff
@@ -1,29 +1,70 @@
 <template>
 	<div id="countdown">
-		{{ timeStamp }}
+		<p class="title">COUNTDOWN!</p>
```

Comment:
in the design, it says `CUTC COUNTDOWN!` and its also slightly smaller than the numbers.